packages/cli/templates/react/src/components/AssetSelector.tsx

```diff
@@ -1,21 +1,45 @@
-import { FunctionComponent, useState } from 'react'
-import { SelectAsset } from '@storyblok/field-plugin'
-import Button from './button/Button'
+import { useState } from 'react'
+import { PluginActions } from '@storyblok/field-plugin'
 
-type AssetSelectorFunc = FunctionComponent<{
-  selectAsset: SelectAsset
-}>
-const AssetSelector: AssetSelectorFunc = ({ selectAsset }) => {
```

Comment:
Removed `FunctionComponent` because of https://github.com/storyblok/smartling-tool-extension/pull/3#discussion_r1151784442

Reply:
I left another reply there: https://github.com/storyblok/smartling-tool-extension/pull/3#discussion_r1161546165

Basically, React 18 removes the child prop from FunctionComponent.

Though this is okay as well. I might add `: JSX.Element` as the function's return type.

Reply:
Oh interesting. Thanks for the info.

Reply:
FYI @BibiSebi